src/sentry/notifications/notifications/rules.py

```diff
@@ -121,7 +130,7 @@ def get_context(self) -> MutableMapping[str, Any]:
             "slack_link": get_integration_link(self.organization, "slack"),
             "notification_reason": notification_reason,
             "notification_settings_link": absolute_uri(
-                "/settings/account/notifications/alerts/?referrer=alert_email"
+                f"/settings/account/notifications/alerts/?referrer=alert_email{fallback_param}"
```

Comment:
@scttcper Nit: Might be better to do a more programmatic way to create the query params so we can add more in the future more easily. Something using `urllib.parse` perhaps.